### DISCUSSION TITLE
使用el-upload组件时，在72版本chrome浏览器，上传会报错，初步定位到源码在解构赋值props时，Proxy的Proxy无法正确解构，造成传入属性失效

### DISCUSSION THREAD

**Discussion by qiugui** [element-plus/element-plus#10723] [Q&A]
使用el-upload组件时，在72版本chrome浏览器，上传会报错，初步定位到源码在解构赋值props时，Proxy的Proxy无法正确解构，造成传入属性失效，不知这个问题是否解决

**Comment by jxl-hash** [comment]
请问解决了吗

**Reply by btea** [reply]
我之前遇到过类似的情况，定位代码发现是在该版本浏览器中`auto-upload`取值时得到的是`true`，从而自动上传导致的报错。我之前的解决方式是在对项目中使用的element-plus版本打补丁，将`autoUpload`默认值改成`false`。

上述方案仅供参考，不确定你遇到的具体情况。如果不是的话，请提供一份简单的复现代码。
